math/minuit2/src/MnSeedGenerator.cxx

```diff
@@ -80,44 +65,36 @@ MinimumSeed MnSeedGenerator::operator()(const MnFcn& fcn, const GradientCalculat
       dcovar = 0.;
    } else {
       for(unsigned int i = 0; i < n; i++)
-         mat(i,i) = (fabs(dgrad.G2()(i)) > prec.Eps2() ? 1./dgrad.G2()(i) : 1.);
+         mat(i,i) = (std::fabs(dgrad.G2()(i)) > prec.Eps2() ? 1./dgrad.G2()(i) : 1.);
    }
    MinimumError err(mat, dcovar);
 
    double edm = VariableMetricEDMEstimator().Estimate(dgrad, err);
    MinimumState state(pa, err, dgrad, edm, fcn.NumOfCalls());
 
-   if (printLevel >1) {
-      MnPrint::PrintState(std::cout, state, "MnSeedGenerator: Initial state:  ");
-   }
+   print.Info("Initial state:", state);
```

Comment:
Before the function MnPrint::PrintState was printing much less information and all in one line, see:
https://github.com/root-project/root/blob/5134568c209d8288ba38e8d31d45744eb79b0863/math/minuit2/src/MnPrint.cxx#L58

Maybe one could re-use this function. 
Wehn running in debug mode maybe is more interesting having the full state information including parameter values which can be a lot !

Reply:
Ok, I will try to restore this.

Reply:
ok, now implemented